src/bpmn-visualization.ts

```diff
@@ -27,8 +27,7 @@ export { StyleConfigurator };
 export { ShapeUtil };
 // end of 'TO DO remove dedicated import/export'
 
-export { BpmnVisualizationOptions, LoadOptions } from './component/BpmnVisualization';
-export { FitType } from './component/mxgraph/Options';
+export { BpmnVisualizationOptions, LoadOptions, FitType } from './component/Options';
```

Comment:
:question: export * for simplicity? all types in this file are in the public API

Reply:
I asked myself the same question ^^
I'll change it